CMakeLists.txt

```diff
@@ -176,6 +180,15 @@ if(VELOX_ENABLE_S3)
   add_definitions(-DVELOX_ENABLE_S3)
 endif()
 
+if(VELOX_ENABLE_GCS)
+  # Set GCS_ROOT_DIR if you have a custom install location of GCS SDK CPP.
+  if(GCSSDK_ROOT_DIR)
+    set(CMAKE_PREFIX_PATH ${GCSSDK_ROOT_DIR})
```

Comment:
Its better if we append , since this will override custom cmake_prefix_paths provided via commandline.

Reply:
Agree, I will fix it.

Reply:
```suggestion
    list(APPEND CMAKE_PREFIX_PATH ${GCSSDK_ROOT_DIR})
```

Reply:
```suggestion
    list(APPEND CMAKE_PREFIX_PATH ${GCSSDK_ROOT_DIR})
```